apps/site/lib/site_web/views/trip_compare_view.ex

```diff
@@ -532,20 +532,8 @@ defmodule SiteWeb.TripCompareView do
 
     transit_legs
     |> Stream.with_index()
-    |> Enum.reduce(0, fn {leg, leg_index}, acc ->
-      if leg_index < 1 do
-        acc + (leg |> Fares.get_fare_by_type(fare_type) |> fare_cents())
-      else
-        # Look at this transit leg and previous transit leg(s)
-        two_legs = transit_legs |> Enum.slice(leg_index - 1, 2)
-        three_legs = transit_legs |> Enum.slice(leg_index - 2, 3)
-        # If this is part of a free transfer, don't add fare
-        cond do
-          Transfer.is_maybe_transfer?(three_legs) -> acc
-          Transfer.is_maybe_transfer?(two_legs) -> acc
-          true -> acc + (leg |> Fares.get_fare_by_type(fare_type) |> fare_cents())
-        end
-      end
+    |> Enum.reduce(0, fn {leg, _leg_index}, acc ->
```

Comment:
Nice! 
Since we're not using the index value, we can actually remove the previous step (`Stream.with_index()`), which is what was adding the index in the first place. Removing `Stream.with_index()` would need to alter this from `fn {leg, _leg_index}, acc ->` to `fn leg, acc ->`.